docker/README.md

```diff
@@ -112,12 +112,12 @@ Now start `pxe-1` from VirtualBox. You should see it boot and auto automatically
     * By default the Vagrant Docker VM exposes all related ports. Some of which are only necessary for development and debugging. You can disable any ports you do not wish to use, or change the which port on the host they map too.
     * Ensure you have the right version of Vagrant for VirtualBox. Later versions of VirtualBox require a more recent version of Vagrant.
 
-## Run a minimal set of RackHD containers (without ELK).
-For who doesn't need ELK running along with RackHD, due to system performance restriction or docker-pull network bandwidth restriction: an alternative docker-compose file can be used by "-f" option:
+## Run RackHD containers (with ELK).
+For who need ELK running along with RackHD, due to system performance restriction or docker-pull network bandwidth restriction: an alternative docker-compose file can be used by "-f" option:
 
```

Comment:
OK, I add one line in README.md. "Open http://127.0.0.1:5601 in a web browser, you can see kibana's website."